ada_feeding/ada_feeding/behaviors/__init__.py

```diff
@@ -2,11 +2,3 @@
 This package contains custom py_tree behaviors for the Ada Feeding project.
 """
 from .blackboard_behavior import BlackboardBehavior
-from .compute_move_to_mouth_position import ComputeMoveToMouthPosition
-from .modify_collision_object import (
-    ModifyCollisionObject,
-    ModifyCollisionObjectOperation,
-)
-from .move_to import MoveTo
-from .move_to_dummy import MoveToDummy
-from .toggle_collision_object import ToggleCollisionObject
```

Comment:
This breaks `egordon/full_schema` as-is, so we should merge this *first* and then fix egordon/full_schema before merging.